README.md

```diff
@@ -1,4 +1,5 @@
 [![CI](https://github.com/cronn/reflection-util/workflows/CI/badge.svg)](https://github.com/cronn/reflection-util/actions)
+[![Valid Gradle Wrapper](https://github.com/cronn/reflection-util/workflows/Validate%20Gradle%20Wrapper/badge.svg)](https://github.com/cronn/reflection-util/actions/workflows/gradle-wrapper-validation.yml)
```

Comment:
I would actually move this badge rather to the end of the list. IMO the Maven Central badge is more important.

Reply:
moved